tests/test_app/test_sample.py

```diff
@@ -499,6 +500,14 @@ def test_omit_bad_seqs(bad_gap_data, gap_fraction, quantile, expected_keys):
     assert got.to_dict() == expected
 
 
+def test_omit_bad_seqs_error():
+    with pytest.raises(
+        MolTypeError,
+        match="Invalid moltype: text. Moltype must be one of DNA, RNA, or PROTEIN.",
```

Comment:
This test is not passing because the error messages do not match. I know that the AI tools recommend checking the error messages, but I disagree. I think they make the test fragile, as this case shows. Delete the match line.